rdr_service/dao/genomics_dao.py

```diff
@@ -3116,6 +3116,46 @@ def get_id(self, obj):
     def from_client_json(self):
         pass
 
+    def get_appointments_gror_revoked(self):
+        max_appointment_id_subquery = sqlalchemy.orm.Query(
+            functions.max(GenomicAppointmentEvent.appointment_id).label(
+                'max_appointment_id'
+            )
+        ).filter(
+            GenomicAppointmentEvent.event_type.notlike('%note_available')
+        ).group_by(
+            GenomicAppointmentEvent.participant_id,
+            GenomicAppointmentEvent.module_type
+        ).subquery()
+
+        max_event_authored_time_subquery = sqlalchemy.orm.Query(
+            functions.max(GenomicAppointmentEvent.event_authored_time).label(
+                'max_event_authored_time'
+            )
+        ).filter(
+            GenomicAppointmentEvent.event_type.notlike('%note_available')
+        ).group_by(
+            GenomicAppointmentEvent.participant_id,
+            GenomicAppointmentEvent.module_type
+        ).subquery()
```

Comment:
So don't think we will need these subqueries here. We just need a query that denotes that the participant has an appointment in the system, but their GROR is now equivalent to "no" or "i'm not sure" via the ticket AC.